tools/simple_hydrog/libfmslite/horiz_interp.F90

```diff
@@ -1,3 +1,22 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL FRE NetCDF tools package (FRE-NCTools).
+!*
+!* FRE-NCTools is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FRE-NCTools is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FRE-NCTools (LICENSE.md) .  If not, see
+!* <http://www.gnu.org/licenses/>.
+!***********************************************************************
```

Comment:
Still have the old license header.  Should this remain an FMS license as the file originated there?

Reply:
Good question. I myself do not know at this time.

Reply:
Revisiting this. It looks like the process I had did not catch this one for removal. I plan to remove the old text since it does not contain the text "Copyright".

Reply:
See https://github.com/NOAA-GFDL/FRE-NCtools/pull/43#discussion_r562767563